packages/terra-site/src/examples/search-field/Index.jsx

```diff
@@ -35,6 +37,11 @@ const SearchFieldExamples = () => (
     <h2 id="searchFieldDelayed">Search Field with delay of 2000ms</h2>
     <SearchFieldDelayed />
     <br />
+    <h2 id="searchFieldDefaultValue">Search Field with a default value</h2>
+    <SearchFieldDefaultValue />
+    <br />
+    <h2 id="searchFieldOnChange">Search Field with custom onChange handler</h2>
```

Comment:
I feel like we don't really need a site example for onChange

Reply:
I went back and forth on whether I should include it or not, there is nothing visually different so I will remove it